controllers/controllers.js

```diff
@@ -0,0 +1,8 @@
+const endpointsJSON = require("../endpoints.json");
+
+function getApi(req, res) {
+  console.log(endpointsJSON, " <==== endpoints");
```

Comment:
Don't leave console logs in completed code. Once the tests pass make sure to remove them.